### DISCUSSION TITLE
Web Interface not reachable

### DISCUSSION THREAD

**Discussion by KrOnAsK** [jomjol/AI-on-the-edge-device#3093] [General]
### The Problem

Hi, I configured the ESP Cam a couple of days ago and everthing worked out fine but today the web interface is not reachable but if I ping it via CMD it works just fine 

### Version

latest 

### Logfile

```shell
I can't access the log files
```

### Screenshots

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92718910/fcc1052c-cac2-44cf-a076-c46733bf2041)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92718910/259148ea-8de0-49df-9715-48650cf9668e)

**Comment by SybexX** [comment]
have you tried the following address?

192.168.178.113/info
192.168.178.113/sysinfo
192.168.178.113/starttime
192.168.178.113/statusflow
192.168.178.113/cpu_temperature
192.168.178.113/rssi
192.168.178.113/uptime
192.168.178.113/value
192.168.178.113/json
192.168.178.113/heap
192.168.178.113/stream

what value does your rssi have, you can normally read it in the router (e.g. Fritzbox)?

**Comment by KrOnAsK** [comment]
Hi, I attached the screenshot from my fritzbox its -65dBm 
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92718910/bf4f3080-07fc-461f-8ff5-5105012bcced)
and I also tried all the links above none worked

**Comment by SybexX** [comment]
have you tried with another pc or did you briefly disconnect the ESP from the power?
Some people have reported that their antivirus software blocked the site, others that they had problems with VPN.

**Comment by freese0711** [comment]
The same with my module.

After a ping the webinterface of the device can be reached for about 10 minutes. Configuration can be done.
After that time is can not be reached anymore but the module works fine. I get values via MQTT.

**Comment by KrOnAsK** [comment]
> have you tried with another pc or did you briefly disconnect the ESP from the power? Some people have reported that their antivirus software blocked the site, others that they had problems with VPN.

Yes I have tried it all but I can't access the website, not with chrome not with firefox not with edge and tried restarting multiple times

**Comment by SybexX** [comment]
Then I would say there is something wrong with your SD card.
Either something has destroyed some files or the SD card is broken(shortly from total failure).
Or what could also be that the ESP32 runs hot and thereby reduces the performance of the WiFi and the CPU.
The heat from the processor and the Camera is also transferred to the SD card, which could also cause it to fail.

**Comment by freese0711** [comment]
> ### The Problem
> Hi, I configured the ESP Cam a couple of days ago and everthing worked out fine but today the web interface is not reachable but if I ping it via CMD it works just fine
> 
> ### Version
> latest
> 
> ### Logfile
> ```shell
> I can't access the log files
> ```
> 
> ### Expected Behavior
> _No response_
> 
> ### Screenshots
> ![image](https://private-user-images.githubusercontent.com/92718910/330386838-fcc1052c-cac2-44cf-a076-c46733bf2041.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTczMDg5NjUsIm5iZiI6MTcxNzMwODY2NSwicGF0aCI6Ii85MjcxODkxMC8zMzAzODY4MzgtZmNjMTA1MmMtY2FjMi00NGNmLWEwNzYtYzQ2NzMzYmYyMDQxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjAyVDA2MTEwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5ZDk4NGU4YTc4ODczZDg3Mzc5ZDkzYmM5N2Y1NWM3MzdlNWNhMzFiYmQ3ZDdjYmJiYzUyMWQ4MWZmZGFmYjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uBA8_u50txZZJYk1Ua8EkJOkv2VKi3i-TqHPHJtNQPU) ![image](https://private-user-images.githubusercontent.com/92718910/330386902-259148ea-8de0-49df-9715-48650cf9668e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTczMDg5NjUsIm5iZiI6MTcxNzMwODY2NSwicGF0aCI6Ii85MjcxODkxMC8zMzAzODY5MDItMjU5MTQ4ZWEtOGRlMC00OWRmLTk3MTUtNDg2NTBjZjk2NjhlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjAyVDA2MTEwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk2ZTkyMTI2ZmM1MTNkZjQyZjE5MGVjZmIwZTIxYzE0ZWVjMjhlNzI4MWVmNGY0MWMyNmE4YTMwNDI2ZTE0OTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Aj0mcRO9B7c4dKlJ_5dhiTw3RqwWtz3ddi8V72KYbvE)
> 
> ### Additional Context
> _No response_

Hallo,
my solution was to Set WLAN Channel 1 and disable Autochannel in FB 7390.

Best regards
Freese
